src/db/commentToDB.php

```diff
@@ -54,21 +60,21 @@ function add_comment_to_db(){
 	//returns a server path to a page
 	function get_redirect_path($value){
 		
-		$p_id = fetch_p_id();
+		$p_id = $this->fetch_p_id();
 		if($p_id == -1){$value = -4;}
 		switch($value){
-		/*no user*/		 case(-3): return "../pages/SignUpPage/signUP.php?source=post";
-		/*no p_id*/		 case(-4): return "../pages/viewPostPage/viewPost.php?id= $p_id&source=noP_id";
-		/*no comment*/   case(-1): return "../pages/viewPostPage/viewPost.php?id= $p_id&source=noComment";
-		/*comment success*/ default: return "../pages/viewPostPage/viewPost.php?id= $p_id";
+		/*no user*/		 case(-3): return "../SignUpPage/signUP.php?source=post";
+		/*no p_id*/		 case(-4): return "../viewPostPage/viewPost.php?id= $p_id&source=noP_id";
+		/*no comment*/   case(-1): return "../viewPostPage/viewPost.php?id= $p_id&source=noComment";
+		/*comment success*/ default: return "../viewPostPage/viewPost.php?id= $p_id";
 		}
-		return "../pages/viewPostPage/viewPost.php?id= $p_id";
+		return "../viewPostPage/viewPost.php?id= $p_id";
```

Comment:
Why did paths change? Files have not moved? Are you sure this change is valid?

Reply:
path didn't work before changes, they are tested and work

Reply:
thanks for verifying